recipes/mox/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: mox
+  version: {{ version }}
+
+source:
+  fn: mox-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/m/mox/mox-{{ version }}.tar.gz
+  md5: 6de7371e7e8bd9e2dad3fef2646f4a43
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - mox
+
+about:
+  home: http://code.google.com/p/pymox/
```

Comment:
I looked, but I didn't see one. Normally, if they do the GitHub export it will at least redirect to the new GitHub repo. That wasn't the case here. Not sure why they haven't moved this one yet, but expect it will happen eventually.